docs/contributing.md

```diff
@@ -24,61 +59,88 @@ Fork and Clone Bulwark
     git remote add upstream https://github.com/ZaxR/bulwark.git
     ```
 
+
 Set up a Development Environment
 --------------------------------
 
-Bulwark supports Python 3.5+. It\'s recommended to use version 3.5 for
-development to ensure newer features aren\'t accidentally used, though
-CI tools will check all versions on the creation of a PR.
+Bulwark supports Python 3.5+. For your local development version of Python it\'s recommended to
```

Comment:
Many cloud environments only support older versions of python. For example, Google's cloud DS environments only support 3.5 in 90% of cases, and only 3.6 in the other 10%. Similarly, other DS libraries like Tensorflow are slow to add support for new versions because of the complexity involved. I'm okay with the overhead to ensure all bulwark's target users can use the library.